### PR TITLE
added Teddy Cash, updated tags, filter option, theming

### PR DESCRIPTION
**Pull request by prodvdc**

### CODE REVIEW

File: contexts/compound-and-earn-context.js [hunk 1]
```diff
@@ -118,11 +118,12 @@ export function CompoundAndEarnProvider({ children }) {
       source: pool.source,
       symbol:
         pool.source === 'Trader Joe' ? 'JLP'
```
Comment (saito-sv) [inline contexts/compound-and-earn-context.js:120]: These values like `TLP` and `BLT` could use a constant instead of literals.

**Review by saito-sv — DISMISSED** [review]
LGTM

File: contexts/compound-and-earn-context.js (hunk 1 above)
Comment (prodvdc) [reply contexts/compound-and-earn-context.js:120]: Yes. @saito-sv 
Can we do this refactoring on the new asana task after merging this PR?

Comment (saito-sv) [reply contexts/compound-and-earn-context.js:120]: yup

**Comment by ghost** [comment]
LGTM

**Review by ghost — APPROVED** [review]

**Pull request merged**